src/utils/string/StringUtils.js

```diff
@@ -398,6 +398,10 @@ const onOpenLink = (url) => {
   });
 };
 
+const isValidUrl = (url) => {
+  return urlRegex.test(url);
+};
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `isValidUrl` function is correctly using the `urlRegex` to test if a given string is a valid URL. Good job on reusing the regex pattern here.



<!-- This is an auto-generated comment by OSS CodeRabbit -->